pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = ""
 authors = ["Gabe <wei.g.chu@gmail.com>"]
 
 [tool.poetry.dependencies]
-python = "3.7"
+python = "^3.7"
 tensorflow = "^2.1.0"
```

Comment:
I think for TensorFlow this might be a bit...wild...especially after the USEM project. I think it would be better to use the Tilde requirement.  what do you think?

Reply:
That makes sense. Will use tilde instead.